File: Easy/capitalize_words.py
data = ['Hello world', 'javaScript language', '1st thing', '4TEST', 'TEST']

#import sys
#test_cases = open(sys.argv[1], 'r')
#data = test_cases.readlines()
for test in data:
	test = test.rstrip()
	words = test.split(" ")
	new_words = []
	for w in words:
		cap_word = w[0].upper() + w[1:]
		new_words.append(cap_word)
	print " ".join(new_words)

#test_cases.close()

File: Easy/capitalize_words.r
test.cases <- c("Hello world", "javaScript language",
				"a letter", "1st thing")

#args <- commandArgs(trailingOnly=TRUE)
#test.cases <- strsplit(readLines(args[[1]], warn=FALSE), '\n')
cat(sapply(test.cases, function(test) {
	if (length(test) > 0) {
		words <- strsplit(test, " ")[[1]]
		words <- sapply(words, function(s){paste(toupper(substring(s, 1, 1)), substring(s, 2), sep = "", collapse = "")})
		paste(words, sep = " ", collapse = " ")
	}
}), sep = "\n")